dipy/tracking/utils.py

```diff
@@ -571,7 +572,7 @@ def target(streamlines, target_mask, affine, include=True):
 
     Raises
     ------
-    IndexError
+    ValueError
```

Comment:
This seems strange because I modified the exception of another function that I haven't modified, but I simply fixed the error. The fonction actually raises an `ValueError`, not a `IndexError`.